src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -593,7 +614,7 @@ PyMODINIT_FUNC PyInit_pythonmonkey(void)
     return NULL;
   }
 
-  if (PyModule_AddObject(pyModule, "SpiderMonkeyError", SpiderMonkeyError)) {
+  if (PyModule_AddObject(pyModule, "SpiderMonkeyError", SpiderMonkeyError) < 0) {
     Py_DECREF(pyModule);
     return NULL;
   }
```

Comment:
this block seems outdated:

  bool rvalIsFunction = cls == js::ESClass::Function;   // function object
  bool rvalIsString = rval.isString() || cls == js::ESClass::String;   // string primitive or boxed String object
  if (!(rvalIsFunction || rvalIsString)) {   // rval may be a JS function or string which must be kept alive.
    delete rval;
  }